src/components/items-table/items-table.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Link from "next/link";
+import Image from "next/image";
+import CurrencyValueDisplay from "../currency-value-display";
+import HSparkline from "../hsparkline";
+import { GeneralUtils } from "../../utils/general-util";
+import { ItemGroupValueTimeseries } from "../../__generated__/resolvers-types";
+
+type ItemsProps = {items: ItemGroupValueTimeseries[], league: string};
+
+export default function ItemsTable({items, league }: ItemsProps) {
```

Comment:
Can probably just leave this as a second class inside the page, I won't be used anywhere else and if it it is they can move it out into its own file.